solr-jsx.js

```diff
@@ -558,7 +558,7 @@ Solr.QueryingURL.prototype = {
   
 
 var paramIsUrlOnly = function(name) {
-  return name.match(/^(json\.nl|json\.wrf|q|wt|start)/);
+  return name.match(/^(json\.nl|json\.wrf|q|wt|start|json2.+)/);
```

Comment:
`<ocd mode>`Would it be better to sort the alternatives? Shouldn't affect the matching, but perhaps looks a bit more orderly, especially if more will need to be added in the future?`</ocd mode>`
```suggestion
  return name.match(/^(json\.nl|json\.wrf|json2.+|q|start|wt)/);
```

Also, my assumption might be wrong, but if we're matching against the parameter name and only the parameter name, would it be better to also match the end of the string, just to be sure it's, e.g., `start` and not `startsomething`, i.e.
```suggestion
  return name.match(/^(json\.nl|json\.wrf|json2.+|q|start|wt)$/);
```

Reply:
You're right. This functionality is borrowed directly from the `SolrJs` library, that was used as a base.